XcodeServerSDK/XcodeServer.swift

```diff
@@ -415,6 +447,32 @@ public extension XcodeServer {
         }
     }
     
+    /**
+    XCS API call for retrievieng all available integrations on server.
+    
+    - parameter integrations:   Optional array of integrations.
+    - parameter error:          Optional error.
+    */
+    public func getIntegrations(completion: (integrations: [Integration]?, error: NSError?) -> ()) {
```

Comment:
@czechboy0 while working on #47 I've noticed thath your method: `getBotIntegrations()` use _filters_. Do you have any clue where I can find available keys for this `:filter?` query? While _Charlesing_ around I've noticed only one:

- `last=`

So I went to `integrationFilterClass.js` but guess what - can't find anything... 😕 Give me some hints as I want to include those to **header docs**.